src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -87,10 +87,9 @@ You'll need:
 In this step, you'll set up an external connection from the destination cluster to the source cluster. Depending on how you manage certificates, you must ensure that all nodes between the clusters have access to the certificate of the other cluster.
 
 You can use the `cockroach encode-uri` command to generate a connection string containing a cluster's certificate.
+LDR from the destination cluster.
```

Comment:
This appeared like a random sentence/not finished

Reply:
Oh weird, will take a look. Thanks for finding!